### PR TITLE
Spring boot practice shopping cart

### PR DESCRIPTION
**Pull request by SerhiiAN**

### CODE REVIEW

File: src/main/java/mate/academy/intro/dto/shoppingcart/CreateCartItemRequestDto.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package mate.academy.intro.dto.shoppingcart;
+
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+
+public record CreateCartItemRequestDto(
+        @NotNull(message = "Book ID is mandatory.")
```
Comment (okuzan) [inline src/main/java/mate/academy/intro/dto/shoppingcart/CreateCartItemRequestDto.java]: for Long ids we can use @Positive

File: src/main/java/mate/academy/intro/dto/shoppingcart/UpdateCartItemRequestDto.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package mate.academy.intro.dto.shoppingcart;
+
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+
+public record UpdateCartItemRequestDto(
+        @NotNull(message = "Quantity is mandatory.")
```
Comment (okuzan) [inline src/main/java/mate/academy/intro/dto/shoppingcart/UpdateCartItemRequestDto.java]: primitive cannot be null

**Review by okuzan — CHANGES_REQUESTED** [review]
please fix validation

File: src/main/java/mate/academy/intro/repository/shopping/cart/ShoppingCartRepository.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package mate.academy.intro.repository.shopping.cart;
+
+import mate.academy.intro.model.ShoppingCart;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
+import org.springframework.stereotype.Repository;
+
+@Repository
```
Comment (MentorMateAcademy) [inline src/main/java/mate/academy/intro/repository/shopping/cart/ShoppingCartRepository.java]: Repository anno is redundant in JPA

**Review by MentorMateAcademy — APPROVED** [review]

**Pull request merged**